src/Sleep.js

```diff
@@ -17,42 +17,24 @@ class Sleep {
     return [...specificUser].splice(-7 * week, 7).map(day => day.date);
   }
 
-  returnAvgSleepHours() {
+  returnAvgSleepInfo(sleepInfo) {
     let specificUser = this.findUser();
-    return Number((specificUser.reduce((totalHours, day) => {
-      totalHours += day.hoursSlept;
-      return totalHours;
+    return Number((specificUser.reduce((acc, day) => {
+      acc += day[sleepInfo];
+      return acc;
     }, 0) / specificUser.length).toFixed(2));
-  }
-
-  returnAvgSleepQuality() {
-    let specificUser = this.findUser();
-    return Number((specificUser.reduce((totalQuality, day) => {
-      totalQuality += day.sleepQuality;
-      return totalQuality;
-    }, 0) / specificUser.length).toFixed(2));
-  }
-
-  returnSleepHours(date) {
-    let specificUser = this.findUser();
-    return specificUser.find(day => day.date === date).hoursSlept;
-  }
-
-  returnSleepQuality(date) {
-    let specificUser = this.findUser();
-    return specificUser.find(day => day.date === date).sleepQuality;
-  }
+  };
 
-  returnWeekOfSleepHours(week) {
+  returnSleepInfo(date, sleepInfo) {
     let specificUser = this.findUser();
-    return this.returnWeekOfData(week, specificUser).map(day => day.hoursSlept);
+    return specificUser.find(day => day.date === date)[sleepInfo];
   }
 
-  returnWeekOfSleepQuality(week) {
+  returnWeekOfSleepInfo(week, sleepInfo) {
     let specificUser = this.findUser();
```

Comment:
I'm about to push up my work on the inherit stuff and I was able to get rid of the specificUser = this.findUser() on pretty much every function using inherit so these functions will be real clean once we combine these!